src/ts/components/dashboard/TopMenu.tsx

```diff
@@ -2,10 +2,11 @@ import * as React from 'react'
 import { connect } from 'react-redux'
 import { withRouter, RouteComponentProps } from 'react-router'
 import { Image, Grid } from 'semantic-ui-react'
-import { ChainDropdown } from '../basic-components'
 import { networks } from '../../constants/networks'
 import { IAppState } from '../../background/store/all'
 import { saveSettings } from '../../background/store/settings'
+import {ChainDropdown} from '../basic-components'
```

Comment:
format seems still not right in your IDE

Reply:
yup